xml/operations-maintenance-database_maintenance.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook51-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="database-maintenance">
+  <title>Periodic OpenStack Maintenance Tasks</title>
+  <para>
+    Heat-manage helps manage Heat specific database operations. The associated
+    database should be periodically purged to save space. The following should
+    be setup as a cron job on the servers where the heat service is running at
+    <literal>/etc/cron.weekly/local-cleanup-heat</literal>
+    with the following content:
+  </para>
+  <screen>
+  #!/bin/bash
+  su heat -s /bin/bash -c "/usr/bin/heat-manage purge_deleted -g days 14" || :
+  </screen>
+  <para>
+     nova-manage db archive_deleted_rows command will move deleted rows
+     from production tables to shadow tables. Including
+     <literal>--until-complete</literal> will make the command run continuously
+     until all deleted rows are archived. It is recommended to setup this task
+     as <literal>/etc/cron.weekly/local-cleanup-nova</literal>
+     on the servers where the nova service is running, with the
+     following content:
+  </para>
+  <screen>
+  #!/bin/bash
+  su nova -s /bin/bash -c "/usr/bin/nova-manage db archive_deleted_rows --until-complete" || :
+  </screen>
+  <para>
+     Fernet tokens should be rotated frequently for security purposes.
+     It is recommended to setup this task as a cron job in
+     <literal>/etc/cron.weekly/openstack-keystone-fernet</literal>
+     on the server where the keystone service is running,
+     with the following content:
+  </para>
+  <screen>
+  #!/bin/bash
+  su keystone -s /bin/bash -c "keystone-manage fernet_rotate"
+
+  /usr/bin/keystone-fernet-keys-push.sh 192.168.81.168; /usr/bin/keystone-fernet-keys-push.sh 192.168.81.169;
```

Comment:
this part is for Crowbar right? For Ardana, the keystone playbooks take care of that I think.

Reply:
that makes sense... I'll see if I can set that to only render in the crowbar version